packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/List.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Font } from '@samuelmeuli/font-manager';
+import { List, SearchFonts, FontLI, FontFamily } from './elements';
+
+function getFontId(fontFamily: string): string {
+  return fontFamily.replace(/\s+/g, '-').toLowerCase();
+}
+
+type Props = {
+  fonts: Font[];
+  onSelection: (e: any) => void;
+  activeFontFamily: string;
+  expanded: boolean;
+  suffix: string;
+};
+
+const FontList = ({
+  fonts,
+  onSelection,
+  activeFontFamily,
+  expanded,
+  suffix,
+}: Props) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const updateSearch = e => setSearchTerm(e.target.value);
+  return (
+    <List expanded={expanded}>
+      <SearchFonts
+        type="text"
+        value={searchTerm}
+        onChange={updateSearch}
+        placeholder="Search Typefaces"
+      />
+      {fonts
+        .filter(f =>
+          f.family.toLowerCase().includes(searchTerm.trim().toLowerCase())
+        )
+        .map(font => {
+          const isActive = font.family === activeFontFamily;
+          const fontId = getFontId(font.family);
+          return (
+            <FontLI key={fontId}>
+              <FontFamily
+                type="button"
+                id={`font-button-${fontId}${suffix}`}
+                className={`font-button ${isActive ? 'active-font' : ''}`}
```

Comment:
```suggestion
                className={`font-button ${isActive ? 'active-font' : ''}`}
                data-is-active={isActive}
```

is more expensive in terms of css selection if you decide to drop 5,000,000 of these in the same page but looks nicer